google/oauth2/service_account.py

```diff
@@ -246,6 +246,29 @@ def with_subject(self, subject):
             subject=subject,
             additional_claims=self._additional_claims.copy())
 
+    def with_claims(self, additional_claims):
```

Comment:
Nah, jwt accepts a different set of arguments here (like audience, etc.). One one hand I know there's some duplicating here, but I'm not sure if there's a good way to refactor without messiness.